datahub-web-react/src/app/shared/tags/TagTermGroup.tsx

```diff
@@ -84,105 +64,16 @@ export default function TagTermGroup({
     entitySubresource,
     highlightText,
     refetch,
+    readOnly,
 }: Props) {
     const entityRegistry = useEntityRegistry();
     const [showAddModal, setShowAddModal] = useState(false);
     const [addModalType, setAddModalType] = useState(EntityType.Tag);
-    const tagsEmpty =
-        !editableTags?.tags?.length &&
-        !uneditableTags?.tags?.length &&
-        !editableGlossaryTerms?.terms?.length &&
-        !uneditableGlossaryTerms?.terms?.length;
-    const [removeTagMutation] = useRemoveTagMutation();
-    const [removeTermMutation] = useRemoveTermMutation();
-    const [tagProfileDrawerVisible, setTagProfileDrawerVisible] = useState(false);
-    const [addTagUrn, setAddTagUrn] = useState('');
-
-    const removeTag = (tagAssociationToRemove: TagAssociation) => {
-        const tagToRemove = tagAssociationToRemove.tag;
-        onOpenModal?.();
-        Modal.confirm({
-            title: `Do you want to remove ${tagToRemove?.name} tag?`,
-            content: `Are you sure you want to remove the ${tagToRemove?.name} tag?`,
-            onOk() {
-                if (tagAssociationToRemove.associatedUrn || entityUrn) {
-                    removeTagMutation({
-                        variables: {
-                            input: {
-                                tagUrn: tagToRemove.urn,
-                                resourceUrn: tagAssociationToRemove.associatedUrn || entityUrn || '',
-                                subResource: entitySubresource,
-                                subResourceType: entitySubresource ? SubResourceType.DatasetField : null,
-                            },
-                        },
-                    })
-                        .then(({ errors }) => {
-                            if (!errors) {
-                                message.success({ content: 'Removed Tag!', duration: 2 });
-                            }
-                        })
-                        .then(refetch)
-                        .catch((e) => {
-                            message.destroy();
-                            message.error({ content: `Failed to remove tag: \n ${e.message || ''}`, duration: 3 });
-                        });
-                }
-            },
-            onCancel() {},
-            okText: 'Yes',
-            maskClosable: true,
-            closable: true,
-        });
-    };
-
-    const removeTerm = (termToRemove: GlossaryTermAssociation) => {
```

Comment:
How one thing I think almost 100% we will face merge back issues due to proposals. Just calling out!